src/Templates/src/templates/maui-lib/Platforms/iOS/PlatformClass1.cs

```diff
@@ -0,0 +1,9 @@
+﻿using System;
+
+namespace MauiLib1
+{
+	// All the code in this file is only included on iOS.
```

Comment:
Doesn't this also get used on MacCatalyst?

Reply:
No, there is a separate MacCatalyst folder for that.